poi-ooxml/src/main/java/org/apache/poi/openxml4j/util/ZipSecureFile.java

```diff
@@ -152,7 +167,6 @@ public ZipSecureFile(String name) throws IOException {
      * @throws IllegalStateException if the zip file has been closed
      */
     @Override
-    @SuppressWarnings("resource")
```

Comment:
any reason for this removal?

Reply:
intellij gives a warning highlighting that it is redundant, so it was removed because it literally isnt necessary. theres nothing even in this function called `resource`